render/tests/test-createFragment.js

```diff
@@ -3,6 +3,8 @@
 var o = require("ospec")
 var domMock = require("../../test-utils/domMock")
 var vdom = require("../../render/render")
+var m = require("../../render/hyperscript")
+m.fragment = require("../../render/fragment")
```

Comment:
Sure you want to add `fragment` to `m` here. Might have side-effects. Why not use it directly?